homeworks/ekaterina.sharova_rogerosha/PokemonProject/pokemons.html

```diff
@@ -0,0 +1,141 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>PokemonProject: Pokemons</title>
+        <link
+            href="./pokemons.css"
+            rel="stylesheet"
+            type="text/css">
+    </head>
+    <body>
+        <header class="upper-header">
+            <div class="ma-logo">
+                <img class="MAlogo" src="MAlogo.png" alt="Masters Academy">
+                <img class="MA" src="ma.png" alt="MA">
+            </div>
+            <ul class="navigation">
+                <li>
+                    <a href="#" class="navigation-item">Pokemons</a>
+                </li>
+                <li>
+                    <a href="./index.html" class="navigation-item">About</a>
+                </li>
+            </ul>
+        </header>
+        <div class="content">
+            <h1 class="headline">Study Project "Pokemons"</h1>
+            <div class="white-rectangle">
+                <div class="card-container">
+                    <div class="card">
+                        <div class="card-header">
```

Comment:
Please add content. Here you can find flex containers layouts. Or use grid
<img width="1264" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/b23a3ef4-feac-4afb-a4d4-77d82df4731b">

Reply:
I had some troubles with this part of the task, but everything must be good at the moment

Reply:
> I had some troubles with this part of the task, but everything must be good at the moment

noted that i can't export that yellow background 😥